### DISCUSSION TITLE
Warning: The active manifest file at (...) has an old format that is being maintained.

### DISCUSSION THREAD

**Discussion by chrisoffner3d** [fonsp/Pluto.jl#1710] [Questions about using Pluto]
After upgrading to Julia 1.7 and doing `using Pluto; Pluto.run()`, I get the following warning;

```
Warning: The active manifest file at `/private/var/folders/52/7c2g08dx6k1fjgr5f5c3h9mr0000gn/T/jl_tzS6eE/Manifest.toml` has an old format that is being maintained.
│ To update to the new format run `Pkg.upgrade_manifest()` which will upgrade the format without re-resolving.
└ @ Pkg.Types /Users/administrator/src/julia/usr/share/julia/stdlib/v1.7/Pkg/src/manifest.jl:287
```

However, when I do run `Pkg.upgrade_manifest()`, as suggested, I get:

```
julia> Pkg.upgrade_manifest()
ERROR: Format of manifest file at `/Users/chrisoffner/.julia/environments/v1.7/Manifest.toml` already up to date: manifest_format == 2.0.0
Stacktrace:
 [1] pkgerror(msg::String)
   @ Pkg.Types /Applications/Julia-1.7.app/Contents/Resources/julia/share/julia/stdlib/v1.7/Pkg/src/Types.jl:68
 [2] upgrade_manifest(ctx::Pkg.Types.Context)
   @ Pkg.API /Applications/Julia-1.7.app/Contents/Resources/julia/share/julia/stdlib/v1.7/Pkg/src/API.jl:1689
 [3] upgrade_manifest()
   @ Pkg.API /Applications/Julia-1.7.app/Contents/Resources/julia/share/julia/stdlib/v1.7/Pkg/src/API.jl:1687
 [4] top-level scope
   @ REPL[5]:1
```

Any idea what I'm doing wrong here? Btw. I also get **[this warning](https://github.com/fonsp/Pluto.jl/discussions/1704)**.

**Comment by fonsp** [comment]
Can you record a video of your screen where you show how and when the warning appears?

**Reply by chrisoffner3d** [reply]
After having wiped and reinstalled my Julia environment several times over a couple of days ago, I now cannot reproduce this anymore. I only get [this one](https://github.com/fonsp/Pluto.jl/discussions/1704) now, but that seems to have been resolved. :)

<img width="1193" alt="Screenshot 2021-12-07 at 14 56 14" src="https://user-images.githubusercontent.com/18167754/145042098-651c5d39-25ab-4bcf-9b0c-f1b9106cab3b.png">

**Reply by fonsp** [reply]
It looks like the warning appeared when you used Pluto in Julia 1.7 to open a notebook that was created with Julia <1.7?

**Reply by chrisoffner3d** [reply]
That’s correct.

**Reply by storopoli** [reply]
Any ideas on how to run `Pkg.upgrade_manifest()` on the hard-coded `Manifest.toml` inside a Julia 1.6 notebook that was upgraded to Julia 1.7?

**Reply by jvigneron** [reply]
> Any ideas on how to run `Pkg.upgrade_manifest()` on the hard-coded `Manifest.toml` inside a Julia 1.6 notebook that was upgraded to Julia 1.7?

To remove the warning when opening an old notebook (1.6) after upgrading julia to 1.7, i've just opened the notebook and run the Pkg.upgrade_manifest() inside a cell (after loading Pkg with "using Pkg").

The error didn't reappeared when reopening this notebook.
